wandb/data_types.py

```diff
@@ -740,6 +740,13 @@ def __init__(self, data_or_path, mode=None, caption=None, grouping=None):
         self._width = None
         self._height = None
         self._image = None
+        
+        self._boxes = None 
+        if boxes: 
+            self._boxes = BoundingBoxes2D(boxes)
+        self._masks = None
+        if masks:
+            self._masks = [ImageMask(m) for m in masks]
```

Comment:
We should check if masks is an array, and turn it into an array if it's not.  Do we really need to support multiple masks per image?  May be simpler to support one mask per image...